OpenEdXMobile/res/values/dimens.xml

```diff
@@ -8,6 +8,13 @@
     <!-- Per Material Design Guidelines -->
     <dimen name="action_bar_icon_size">24dp</dimen>
 
+    <!-- Alert dialog default padding -->
+    <dimen name="alert_dialog_padding_width">22dp</dimen>
+    <dimen name="alert_dialog_padding_height">11dp</dimen>
```

Comment:
https://material.google.com/components/dialogs.html#dialogs-specs 

Why not use the Material guideline specs?